### PR TITLE
correcting figure's names

### PR DESCRIPTION
**Pull request by MichalGQM**
correcting the names of the figures

### CODE REVIEW

File: Tutorials/intro-to-timing/readme.md [hunk 1]
```diff
@@ -25,29 +25,29 @@ The script presents five examples in order to demonstrate the timing in QUA.
 The examples and the figures of each example are described below:
 
 1. Two pulses from different elements.
-![timing_in_qua](timing_example1.png "Two pulses from different elements")
+![Two_pulses_from_different_elements](timing_example1.PNG "Two_pulses_from_different_elements")
```
Comment (yomach) [inline Tutorials/intro-to-timing/readme.md:28]: You don't need a title, and also no need for underscore. I would write this as:
`Two pulses from different elements`

And the same goes for all of the others

Comment (MichalGQM) [reply Tutorials/intro-to-timing/readme.md:28]: I saw in intro-to-demod that is was written with underscore so I thought maybe it was a problem as well... 
What is actually the purpose of the sentence written in [ ] ? 
But yes I will just change it and push it...

Comment (yomach) [reply Tutorials/intro-to-timing/readme.md]: This is the alt text that appear wedged you hover on the image...

Google markdown images, markdown syntax or how to write readme.md github to learn the syntax. It's much faster than trying to guess it...

Comment (MichalGQM) [reply Tutorials/intro-to-timing/readme.md]: ok, I changed it and did push... what else is missing?

Comment (yomach) [reply Tutorials/intro-to-timing/readme.md]: nothing, I approved this PR.
You need to resolve this conversation and merge

Comment (MichalGQM) [reply Tutorials/intro-to-timing/readme.md]: ok :)

**Pull request merged**